internal/pkg/manifest/worker_svc_test.go

```diff
@@ -410,6 +410,127 @@ func TestWorkerSvc_ApplyEnv(t *testing.T) {
 			},
 		},
 	}
+	duration111Seconds := 111 * time.Second
+	mockWorkerServiceWithSubscribeNilOverride := WorkerService{
```

Comment:
I feel like it may be worthwhile to also add tests specifically for `Topics` field. Manifest field of type list is always the ones that' are error-prone when it comes to environment override 😭. 

We have had issues with list fields many times (one PR that fixes an issue with a list of security groups: https://github.com/aws/copilot-cli/pull/2442, another PR that fixes an issue with entrypoint which is a list of strings: https://github.com/aws/copilot-cli/pull/2490)

I think since you are using `*[]string` instead of `[]string` it should be fine. We can just add the unit tests to make sure.

Reply:
Added!